src/app/utils/pool-utils.ts

```diff
@@ -174,6 +174,14 @@ export class PoolUtils {
         return "cloud";
     }
 
+    public static getEndOfLifeHyperlink(imageName: string): string {
+        if (imageName.toLowerCase().includes("with-containers")) {
+            return 'https://github.com/Azure/Batch/issues/136';
+        } else {
+            return "";
```

Comment:
This is largely opinion, but IMO it's clearer if this function returns null or undefined when it doesn't find a link. That way Typescript will make consuming code handle the possibility of a null/undefined value instead of having to know to handle empty string (which Typescript can't help with).

Reply:
Updated